### PR TITLE
maintain: update readme and install on k8s docs

### PR DESCRIPTION
**Pull request by technovangelist**
## Summary

This updates the readme and the install on kubernetes docs based on the changes made in the quickstart

## Checklist

- [x] Wrote appropriate unit tests
- [x] Considered security implications of the change
- [x] Updated associated docs where necessary
- [x] Updated associated configuration where necessary
- [x] Change is backwards compatible if it needs to be (user can upgrade without manual steps?)
- [x] Nothing sensitive logged
- [x] Considered data migrations for smooth upgrades

## Related Issues

Resolves #

### CODE REVIEW

**Review by jmorganca — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -29,12 +29,38 @@ Infra manages access to infrastructure such as Kubernetes, with support for [mor
...
+
+```yaml
+server:
+  config:
+    users:
+      - name: admin@example.com
+        password: SetThisPassword! #note this password is now set as plaintext in this file
+
+  # Create a "admin@example.com" user and set a password passed in as a file. The file will need
+  # to be mounted into the pod using `volumes` and `volumeMounts`.
```
Comment (BruceMacD) [inline README.md:42]: Is there a command that allows me to do this easily? How about a link?

File: README.md [hunk 2]
```diff
@@ -29,12 +29,38 @@ Infra manages access to infrastructure such as Kubernetes, with support for [mor
...
+```
+
+
 Install Infra via `helm`:
 
 ```
 helm repo add infrahq https://helm.infrahq.com
 helm repo update
-helm install infra infrahq/infra
+helm update --install infra infrahq/infra
```
Comment (BruceMacD) [inline README.md:63]:
```suggestion
helm upgrade --install infra infrahq/infra -f values.yaml
```

File: docs/install/install-on-kubernetes.md [hunk 3]
```diff
@@ -12,12 +12,36 @@ position: 2
...
+        role: admin
+        resource: infra
+```
 Install Infra via `helm`:
 
 ```
 helm repo add infrahq https://helm.infrahq.com
 helm repo update
-helm install infra infrahq/infra
+helm update --install infra infrahq/infra
```
Comment (BruceMacD) [inline docs/install/install-on-kubernetes.md:44]:
```suggestion
helm upgrade --install infra infrahq/infra -f values.yaml
```

**Review by BruceMacD — COMMENTED** [review]
The `uilogin.png` picture should be updated to this one:
https://github.com/infrahq/infra/blob/brucemacd/self-hosted-docs/docs/images/welcome.png?raw=true

**Pull request merged**